src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Square extends Shape {
+    private final double side;
+
+    public Square(double side, Colour colour) {
+        this.side = side;
+        this.setColour(colour);
+    }
+
+    public void draw() {
+        System.out.println("Drawing square");
+    }
+
```

Comment:
let's use @Override annotation with methods that we override from the abstract class it will be more informative